compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -35,6 +35,12 @@ static llvm::cl::opt<bool> clCheckIRBeforeLLVMConversion(
                    "before conversion to LLVM IR"),
     llvm::cl::init(true));
 
+static llvm::cl::opt<bool> clCheckLinalgVectorized(
+    "iree-codegen-check-linalg-vectorized",
```

Comment:
I reserved `iree-codegen-` for pass names that are in `compiler/Codegen/Common`. Can we just make this `iree-check-linalg-vectorized`?

Reply:
Got it. I replace `iree-codegen-` with `iree-llvmcpu-`. Also update the flag above.